src/utils/got/got.ts

```diff
@@ -0,0 +1,90 @@
+enum METHOD {
+    GET = 'GET',
+    POST = 'POST',
+    PUT = 'PUT',
+    PATCH = 'PATCH',
+    DELETE = 'DELETE',
+}
+
+type Options = {
+    method?: string;
```

Comment:
Тут можно конкретизировать

```suggestion
    method?: METHOD;
```